src/models/workspace.ts

```diff
@@ -53,12 +69,19 @@ export const WorkspaceModel = types
         }
       },
 
-      toggleComparisonVisible(override?: boolean) {
+      toggleComparisonVisible(override?: boolean, muteLog = false) {
```

Comment:
Multiple boolean arguments are somewhat inscrutable at point of call -- would an object with named fields make sense here?, i.e. something like
```typescript
export interface ToggleArgs {
  override?: boolean;
  muteLog?: boolean;
}
toggleComparisonVisible(args?: ToggleArgs)
```

Reply:
I had kind of felt that the IDE makes it clear, and liked how rarely we needed to do that in this project, and have always felt it odd when one random method needs an object argument, but I can see your point.

Is there a way to make an optional object argument, which has a default value if one isn't passed in in Typescript?

```
export interface ToggleArgs {
  override?: boolean;
  muteLog?: boolean;
}
toggleComparisonVisible(args?: ToggleArgs)
```
doesn't set a default for `muteLog`, you need to look in the body of the function to see if one gets set.

```
toggleComparisonVisible({override, muteLog = false}: {override?: boolean; muteLog?: boolean})
```
seems better, but now requires `toggleComparisonVisible` to be called with an argument, so `toggleComparisonVisible()` isn't allowed.
```
toggleComparisonVisible({override, muteLog = false}?: {override?: boolean; muteLog?: boolean})
```
isn't allowed in Typescript. (`A binding pattern parameter cannot be optional in an implementation signature.`)

Reply:
Never mind, got it:

```
toggleComparisonVisible({override, muteLog = false}: {override?: boolean; muteLog?: boolean} = {})
```